java/base-client/src/main/java/no/bankaxept/epayment/client/base/Response.java

```diff
@@ -0,0 +1,11 @@
+package no.bankaxept.epayment.client.base;
+
+public enum Response {
```

Comment:
I think `RequestStatus` would be a better name. The typed responses come as callbacks.

Reply:
It still represents a response though..? I still think `request` is misleading

Reply:
I called it ResponseStatus now..